aspnetcore/client-side/libman/samples/LibManSample/LibManSample.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
The file doesn't have our usual (template) spacing (root element and between the groups).

Reply:
For some reason, I thought I'd display the *.csproj file. I never did, so I'll restore the spacing.